netket/variational.py

```diff
@@ -26,3 +26,58 @@ def _Vmc_iter(self, n_iter=None, step_size=1):
 
 
 Vmc.iter = _Vmc_iter
+
+
+# Higher-level VMC functions:
+
+
+def estimate_expectations(
```

Comment:
this function is not used anymore, to be removed

Reply:
I think it could be useful for end users. For instance, if you have a saved machine and want to compute expectation values in a separate script. As end user, you would have to call `compute_samples`, `local_values`, and `statistics` instead of just doing
```
ma.load("some_file.dat")
sampler = MetropolisLocal(ma)
stats = estimate_expectations([op1, op2], sampler, 50000)
```

Reply:
Ok, I agree good point